src/components/index.js

```diff
@@ -27,6 +27,7 @@ export {
   ModalBody
 } from './Modal';
 
+export { default as TextField } from './TextField';
```

Comment:
You need to add some ts typings in `forms.d.ts`.  AFAIK you are using typescript in the HelpDesk so it could be pretty useful :)

Reply:
Right! Done.